test/Libraries/Microsoft.Extensions.AI.Integration.Tests/ChatClientIntegrationTests.cs

```diff
@@ -341,6 +341,39 @@ public virtual async Task FunctionInvocation_NestedParameters()
         AssertUsageAgainstActivities(response, activities);
     }
 
+    [ConditionalFact]
+    public virtual async Task FunctionInvocation_ArrayParameter()
+    {
+        SkipIfNotEnabled();
+
+        var sourceName = Guid.NewGuid().ToString();
+        var activities = new List<Activity>();
+        using var tracerProvider = OpenTelemetry.Sdk.CreateTracerProviderBuilder()
+            .AddSource(sourceName)
+            .AddInMemoryExporter(activities)
+            .Build();
+
+        using var chatClient = new FunctionInvokingChatClient(
+            new OpenTelemetryChatClient(_chatClient, sourceName: sourceName));
```

Comment:
Is any of the otel stuff necessary for this test?

Reply:
Probably not, I just copied it over from the related test above.